<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaytracingEngineGUIClass</class>
 <widget class="QMainWindow" name="RaytracingEngineGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RaytracingEngineGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="imageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Scene file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sceneFileBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start rendering</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Resolution X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="resolutionXBox">
         <property name="text">
          <string>300</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Resolution Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="resolutionYBox">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RaytracingEngineGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>RaytracingEngineGUIClass</receiver>
   <slot>onStartButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>RaytracingEngineGUIClass</receiver>
   <slot>onBrowseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartButtonClicked()</slot>
  <slot>onBrowseButtonClicked()</slot>
 </slots>
</ui>
